base/remote/DESCRIPTION

```diff
@@ -22,10 +22,9 @@ Imports:
     jsonlite,
     urltools
 Suggests:
-    getPass,
     testthat,
     tools,
-    withr
+    getPass
```

Comment:
I'm skeptical of this change, which removes `withr` and moves thing out of alphabetical order. Are you accidently undoing someone else's fixes?

Reply:
That was changed automatically.